File: API/Core/Entities/VelicinaDresa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Core.Entities
{
    public class VelicinaDresa : BaseEntity
    {
        [Key]
        public Guid VelicinaDresaId { get; set; }

        [Required]
        [StringLength(5)]
        public string VelicinaDresaVrednost { get; set; }

        public int Kolicina { get; set; }
        public Dres Dres { get; set; }
        public Guid DresId { get; set; }

        [JsonIgnore]
        public StavkaPorudzbine StavkaPorudzbines { get; set; }
    }
}

File: Frontend/ClientApp/src/app/shared/models/velicina-dresa.ts
import { Dres } from "./dres";

export class VelicinaDresa {
    velicinaDresaId: string;
    velicinaDresaVrednost: string;
    kolicina: number;
    dres: Dres;
    dresId: string;
  
    constructor(
      velicinaDresaId: string,
      velicinaDresaVrednost: string,
      kolicina: number,
      dres: Dres,
      dresId: string
    ) {
      this.velicinaDresaId = velicinaDresaId;
      this.velicinaDresaVrednost = velicinaDresaVrednost;
      this.kolicina = kolicina;
      this.dres = dres;
      this.dresId = dresId;
    
    }
  }

  export class VelicinaDresaCreation {
   
    velicinaDresaVrednost: string;
    kolicina: number;
  
    dresId: string;
  
    constructor(
   
      velicinaDresaVrednost: string,
      kolicina: number,

      dresId: string
    ) {
   
      this.velicinaDresaVrednost = velicinaDresaVrednost;
      this.kolicina = kolicina;

      this.dresId = dresId;
    
    }
  }

  export class VelicinaDresaUpdate {
   velicinaDresaId: string
    velicinaDresaVrednost: string;
    kolicina: number;
  
    constructor(
      velicinaDresaId: string,
      velicinaDresaVrednost: string,
      kolicina: number,


    ) {
      this.velicinaDresaId=velicinaDresaId
      this.velicinaDresaVrednost = velicinaDresaVrednost;
      this.kolicina = kolicina;


    
    }
  }